src/components/profile-page/settings/profile-change-password.tsx

```diff
@@ -5,11 +5,17 @@
  */
 
 import type { ChangeEvent, FormEvent } from 'react'
-import React, { useState } from 'react'
+import React, { useCallback, useState } from 'react'
 import { Button, Card, Form } from 'react-bootstrap'
 import { Trans, useTranslation } from 'react-i18next'
 import { changePassword } from '../../../api/me'
+import { showErrorNotification } from '../../../redux/ui-notifications/methods'
 
+const REGEX_VALID_PASSWORD = /^[^\s].{5,}$/
```

Comment:
Is 5 a good minium here? Should we enforce a minium anyway? And if we do shouldn't this be something to be configurable?

Reply:
If it's configurable, we probably won't be able to give a good error message…

Reply:
I created an issue to discuss this further: hedgedoc/hedgedoc#1846.